src/codegen.cpp

```diff
@@ -5493,9 +5493,7 @@ static std::unique_ptr<Module> emit_function(
     AttrBuilder *attr = new AttrBuilder();
     attr->addStackAlignmentAttr(16);
 #if JL_LLVM_VERSION >= 50000
-    f->addAttributes(AttributeList::FunctionIndex,
-        AttributeList::get(f->getContext(),
-            AttributeList::FunctionIndex, *attr));
```

Comment:
Is this not still applicable to LLVM 5.0?

Reply:
I think it was just never tested.